src/goto-programs/goto_functions.h

```diff
@@ -18,6 +18,7 @@ Date: June 2003
 
 #include <util/cprover_prefix.h>
 
+/// A set of goto functions
```

Comment:
I'd talk about a map

Reply:
Can I say "a collection of goto functions"? If we say set or map it could seem as if `goto_functionst` itself is a `std::map` or `std::set`.

Reply:
Makes sense!